doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -974,13 +972,28 @@ The commands to fine-tune the reduction strategies and the lazy conversion
 algorithm are described in this section.  Also see :ref:`Args_effect_on_unfolding`,
 which supports additional fine-tuning.
 
+:gdef:`Opaqueness <opaque>` is used to control whether constants can be
+:term:`unfolded <unfold>` with :term:`delta-reduction`.  Opaque means not to
+do unfolding is some cases, while :gdef:`Transparent <transparent>` permits
+unfolding.  Rocq has multiple notions of opaque:
+
+- **Permanently opaque.**  Theorems ending with :cmd:`Qed` are permanently
```

Comment:
Don't introduce yet another terminology for this concept. We struggled to converge on something meaningful. You should pick the terminology used in #19029 already, otherwise the doc is going to end up being more confusing that it is today.

Reply:
Got it.